airflow/cli/commands/celery_command.py

```diff
@@ -125,7 +125,10 @@ def worker(args):
     }
 
     if conf.has_option("celery", "pool"):
-        options["pool"] = conf.get("celery", "pool")
+        pool = conf.get("celery", "pool")
+        options["pool"] = pool
+        from celery import maybe_patch_concurrency
+        maybe_patch_concurrency(['-P', pool])
```

Comment:
Should we leave here a short note of why we are doing this?

Reply:
Good idea!